vos/vos/commands/vcp.py

```diff
@@ -38,7 +44,10 @@
 RA/DEC regions accessed vcp vos:Node/filename.fits(RA, DEC, RAD)
 where RA, DEC and RAD are all given in degrees
 
-Wildcards in the path or filename work also:
+For inventory storage, wildcards in fielname work:
```

Comment:
Misspelled `filename`.  The _proper_ name, too, is Storage Inventory, not inventory storage.

Reply:
Good catch. Removed the entire paragraph as wildcards are not supported, at least for now.